clusterloader2/testing/list/config.yaml

```diff
@@ -83,3 +86,35 @@ steps:
     params:
       namePrefix: "list-configmaps-"
       replicas: 0
+
+- name: Create pods
+  phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+    tuningSet: Sequence
+    replicasPerNamespace: {{$podsNumber}}
+    objectBundle:
+    - basename: {{$podGroup}}
+      objectTemplatePath: pod.yaml
+      templateFillMap:
+        bytes: {{$podBytes}}
+        group: {{$podGroup}}
+
+- module:
+    path: modules/list-benchmark.yaml
+    params:
+      namePrefix: "list-pods-"
+      replicas: {{$listReplicas}}
+      uri: /api/v1/pods?resourceVersion=0
+      namespaced: false
+      contentType: {{$contentType}}
+- module: *startMeasurements
+- name: Wait 5 minutes
+  measurements: *waitMeasurements
+- module: *gatherMeasurements
```

Comment:
Doing measurements this way will override file with configmap measurements. We need to put them in separate files.

cc @mborsz for ideas.